### PR TITLE
Feat/transtate

### PR DESCRIPTION
**Pull request by enieber**
- translate in lib
- translate readme
- remove images
- add lib icons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,38 @@
...
-- package.json添加依赖并安装 / npm i xcc-pdf-editor
-- 解压node_modules中xcc-pdf-editor下的xcc-pdf-editor.7z到项目根目录的public下，xcc-pdf-editor.7z中为资源文件。
-- main.js 引入并 Vue.use
-- Webpack配置中指定makeTextPDF.min.js的加载器为file-loader
-``` js
-    {
-          test: /makeTextPDF\.min\.js$/,
-          loader: 'file-loader'
-    }
-```
```
Comment (vitormattos) [inline README.md:24]: I think that is a very nice idea to have a GitHub page as example of this project and we can replace the Chinese code by English.

File: lib/xcc-pdf-editor.common.js [hunk 2]
```diff
@@ -21197,7 +21197,7 @@ ___CSS_LOADER_EXPORT___.push([module.id, ".whitespace-no-wrap[data-v-5c91f3d0]{w
 
 /***/ }),
 
-/***/ 697:
```
Comment (vitormattos) [inline lib/xcc-pdf-editor.common.js:21200]: Sounds that this is a file that don't need to be versioned.

**Review by vitormattos — COMMENTED** [review]
Have min files that I think is out of context of this PR.
Maybe would be good to create an issue to remove the min files and implement a build/release proccess

**Comment by giovannism20** [comment]
Hey @enieber can u resolve conflict files ? Also is it really ready to be reviewed ?

**Comment by giovannism20** [comment]
Can we close this pull request ?
cc: @vitormattos @enieber

**Comment by vitormattos** [comment]
no more necessary. I made other fixes at project and I don't used the `rollup` because this package sounds very complex to me to build the demo site and also to configure. I used the native vue package `@vue/cli-service` to build the project.

**Pull request closed without merging**